assignments/callbacks.js

```diff
@@ -24,28 +26,74 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
+// ------------------------------------------------------------------
+
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+getLength(items, function(length){
+console.log(length);
+});
+
+
+// ------------------------------------------------------------------
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(items[arr.length-1]);
 }
 
+last(items, function(lastItem){
+  console.log(lastItem);
+})
+
+// ------------------------------------------------------------------
+
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y);
 }
 
+sumNums(3, 3, function(sum){
+  console.log(sum);
+});
+
+// ------------------------------------------------------------------
+
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y);
 }
 
+
+multiplyNums(3, 3, function(product){
+  console.log(product);
+});
+
+// ------------------------------------------------------------------
+
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  list.filter((item) => {
+     if (item === "Pencil"){
```

Comment:
You've hard coded 'Pencil' and locked your code into only caring about Pencil. Is there a better way to make your code more malleable?